p2p/message.go

```diff
@@ -419,3 +419,64 @@ func (tm *TransactionMessage) Id() string {
 	}
 	return hash.String()
 }
+
+// ConsensusMessage is mostly opaque to us
+type ConsensusMessage struct {
+	// Identifies consensus engine.
+	ID uint32
```

Comment:
could you change this to `types.ConsensusEngineId`? from core/types

Reply:
You are right, it makes more sense to re-use the type types.ConsensusEngineId, `ConsensusEngineId` is also the name on substrate code, however in `go` the recommendation is to use uppercase for the Initialisms, check: https://github.com/golang/go/wiki/CodeReviewComments#initialisms
so I updated it to `types.ConsensusEngineID` , hope you agree with the standard, I also saw that the BabeEngineId was not upper so I updated it to upper as well.